packages/api-gateway/src/drivers/http/routes/user/index.js

```diff
@@ -0,0 +1,8 @@
+const { userAdapter } = require('../../adapters');
+const { createUserSchema } = require('./schema');
+
+async function createUserRouter(fastify) {
+  await fastify.post('/', { createUserSchema }, userAdapter.createUser);
```

Comment:
The defined schema does not work because it is not configured correctly, Can you fix it?
```suggestion
  await fastify.post('/', { schema: createUserSchema }, userAdapter.createUser);
```

Reply:
thanks for noticing it, I didn't know it was used like that, it's already corrected.